### PR TITLE
Staging frontend settings

### PR DESCRIPTION
**Pull request by thisconnect**
Screen Shot 2020-08-20 at 9 51 02 AM

Screen Shot 2020-08-20 at 9 57 00 AM

### CODE REVIEW

File: frontends/web/src/routes/settings/settings.tsx [hunk 1]
```diff
@@ -314,24 +314,28 @@ class Settings extends Component<Props, State> {
                                                             ))
                                                         }
                                                     </div>
-                                                </div>
-                                                <div className="column column-1-3">
-                                                    <div class="subHeaderContainer withToggler">
+
+                                                    {/* <div class="subHeaderContainer withToggler">
```
Comment (thisconnect) [inline frontends/web/src/routes/settings/settings.tsx:318]: oups forgot to delete... one sec

**Comment by benma** [comment]
Ethereum should be active if an ERC20 token is active, because the fees are paid in Ether. The master switch was quite neat for that :thinking: I see that the ERC20 tokens are still disabled once ETH is disabled, so I guess it works.

If the grouping is gone, each erc20 token should have the BitBox02-Multi badge.

The expert settings being roughly 1.3 columns wide looks off to me - but that could just be me ;)

**Comment by thisconnect** [comment]
> Ethereum should be active if an ERC20 token is active, because the fees are paid in Ether. The master switch was quite neat for that 🤔 I see that the ERC20 tokens are still disabled once ETH is disabled, so I guess it works.
> 
> If the grouping is gone, each erc20 token should have the BitBox02-Multi badge.
> 
> The expert settings being roughly 1.3 columns wide looks off to me - but that could just be me ;)

We can also just take the first 3 commits and leave the activeaccounts re-aragement for now.

**Comment by thisconnect** [comment]
dang I forgot https://github.com/digitalbitbox/bitbox-wallet-app/pull/977#discussion_r473119522 :)

**Comment by benma** [comment]
> dang I forgot [#977 (comment)](https://github.com/digitalbitbox/bitbox-wallet-app/pull/977#discussion_r473119522) :)

https://github.com/digitalbitbox/bitbox-wallet-app/pull/978/ also touches settings.tsx, changing how the expert fee is stored.

**Comment by benma** [comment]
@x1ddos github CI seems to have a cache problem. Do you think it is a config issue on our side?

I ran golangci locally and it passes, so not a blocker for merging.

**Comment by x1ddos** [comment]
> @x1ddos github CI seems to have a cache problem. Do you think it is a config issue on our side?
> 
> I ran golangci locally and it passes, so not a blocker for merging.

I don't believe it's in our cache. It's one step which we don't configure caching ourselves. They do; no caching here: https://github.com/digitalbitbox/bitbox-wallet-app/blob/a03b641615a2c717ba5589e34c03bd81ecd47188/.github/workflows/ci.yml#L33-L37

Looking for hints in https://github.com/golangci/golangci-lint-action at the moment...

**Comment by x1ddos** [comment]
Actually, doesn't look like cache is the problem. The actual error is:

wallet-app/bitbox-wallet-app/frontends/qt/server/server.go:40:8: could 
not import C (cgo preprocessing failed)]"
   level=error msg="Running error: buildir: analysis skipped: errors in 
package: 
[/home/runner/work/bitbox-wallet-app/bitbox-wallet-app/frontends/qt/server/server.go:40:8: 
could not import C (cgo preprocessing failed)]"

**Comment by x1ddos** [comment]
> Actually, doesn't look like cache is the problem. The actual error is: wallet-app/bitbox-wallet-app/frontends/qt/server/server.go:40:8: could not import C (cgo preprocessing failed)]" level=error msg="Running error: buildir: analysis skipped: errors in package: [/home/runner/work/bitbox-wallet-app/bitbox-wallet-app/frontends/qt/server/server.go:40:8: could not import C (cgo preprocessing failed)]"
> […](#)
> On 8/20/20 1:30 PM, alex wrote: > @x1ddos github CI seems to have a cache problem. Do you think it is a config issue on our side? > > I ran golangci locally and it passes, so not a blocker for merging. I don't believe it's in our cache. It's one step which we don't configure caching ourselves. They do; no caching here: https://github.com/digitalbitbox/bitbox-wallet-app/blob/a03b641615a2c717ba5589e34c03bd81ecd47188/.github/workflows/ci.yml#L33-L37 Looking for hints in https://github.com/golangci/golangci-lint-action at the moment...

Which means github most likely removed gcc from their runners. Would make sense because we didn't change any CI configuration.

**Comment by x1ddos** [comment]
Oh well. Mystery about failing golangci-lint is clear now. It started running on go1.15 for some reason:

```
...
  GOROOT="/opt/hostedtoolcache/go/1.15.0/x64"
  GOTOOLDIR="/opt/hostedtoolcache/go/1.15.0/x64/pkg/tool/linux_amd64"
```
```

And I've verified locally go1.15 results in troubles with our cgo usage. Can't even build qt-linux. Need to make it go back to 1.14 somehow.

**Pull request closed without merging**